docs/reference/search/rrf.asciidoc

```diff
@@ -110,9 +112,58 @@ RRF does not currently support:
 * <<request-body-search-explain, explain>>
 * <<profiling-queries, profiling>>
 
-Using unsupported features as part of a search using RRF will result
+Using unsupported features as part of a search using RRF results
```

Comment:
"a search using RRF" => "a search with RRF" as there are two "using" in this sentence?

Reply:
Changed.